modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -3967,6 +3972,20 @@
 		end
 	end
 
+	function m.linuxMultiProcNumber(cfg)
+		-- Linux equivalent of 'MultiProcessorCompilation'
+		-- Default to 8 parallel jobs
+		if cfg.flags.MultiProcessorCompile then
+			m.element("MultiProcNumber", nil, "8")
+		end
+	end
+
+	function m.linuxLinkTimeCodeGeneration(cfg)
```

Comment:
I have a few questions:
1. Was there any reason to not extend the `linkTimeCodeGeneration` function instead?
2. Is this Linux only, or does Android support it too?
3. The API and element are both "link time optimization", why opt for "link time code generation"?

Reply:
1. This is a good question. On Linux, for whatever reason, the LTO flag is applied to the compiler settings and not to the linker settings, and therefore I need to apply them in a different function. I'm assuming those somehow get passed later to the linker
2. I took a look at Android (on VS2019 as 2022 support for Ant projects is deprecated) and this flag isn't available that I could see
3. I opted for this name because the original function we use is called like that, linkTimeCodeGeneration. And that one is called like that because the vcxproj XML tag is called like that. I can change them all for consistency with the public premake api if we'd rather do that.

Reply:
I'm happy with the answers you've given, going to resolve the conversation.